hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -84,7 +84,7 @@ contracts.scheduleThrottleMaxGasLimit=5000000
 contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
 contract.storageSlotPriceTiers=0til100M,2000til450M
 contracts.throttle.throttleByGas=true
-contracts.precompile.atomicCryptoTransfer.enabled=false
+contracts.precompile.atomicCryptoTransfer.enabled=true
```

Comment:
Turning this on in this pr as it is the only place where this was false instead of creating a new pr.

Reply:
Which envs would this affect?

Reply:
My understanding is mainnet, testnet and previewnet.  Since previewnet configuration is used for local node, it will be turned on there as well.